Src/IronPython.Modules/mmap.cs

```diff
@@ -869,6 +1132,25 @@ internal Bytes GetSearchString() {
                 }
             }
 
+            [SupportedOSPlatform("linux"), SupportedOSPlatform("macos")]
+            private static long GetFileSizeUnix(SafeFileHandle handle) {
+                long size;
+                if (handle.IsInvalid) {
+                    throw PythonOps.OSError(PythonExceptions._OSError.ERROR_INVALID_HANDLE, "Invalid file handle");
+                }
+
+                if (Mono.Unix.Native.Syscall.fstat((int)handle.DangerousGetHandle(), out Mono.Unix.Native.Stat status) == 0) {
+                    size = status.st_size;
+                } else {
+                    Mono.Unix.Native.Errno errno = Mono.Unix.Native.Stdlib.GetLastError();
```

Comment:
Can't use the `PythonNT.GetLastUnixError` here?

Reply:
`PythonNT.GetLastUnixError` cannot be used here (yet) because it does not have proper platform guards (yet). Adding the platform guards to `PythonNT.GetLastUnixError` has a cascading effect to the code. I am saving it for a separate PR (soon), to to mix up too much with this one.